src/main/java/net/mcreator/ui/component/JColor.java

```diff
@@ -46,7 +46,7 @@ public JColor(Window window) {
 
 	public JColor(Window window, boolean allowNullColor) {
 		setLayout(new BorderLayout(2, 0));
-		fl1.setText("255,255,255");
+		fl1.setText("255,255,255,255");

```

Comment:
I'd revert this file as this color chooser is mostly used for Minecraft colors which don't support transparency, so it isn't relevant. It's also not a part of the image maker which makes it out of scope for this PR.